scipy/stats/_continuous_distns.py

```diff
@@ -5195,6 +5195,156 @@ def _entropy(self, beta):
 halfgennorm = halfgennorm_gen(a=0, name='halfgennorm')
 
 
+class breit_wigner_gen(rv_continuous):
+    """
+    Probability density function for relativistic Breit-Wigner distribution [1]_.
+    
+    References
+    ----------
+    .. [1] https://en.wikipedia.org/wiki/Relativistic_Breit-Wigner_distribution
+    
+    Notes
+    -----
+    
+    Plot a Breit-Wigner distribution and random samples
+    ---------------------------------------------------
+
+    >>> import matplotlib.pyplot as plt
+    >>> BW = breit_wigner(mass=125., width=10.)
+    
+    >>> mass = np.linspace(0., 200., 500)
+    >>> pdf = [BW.pdf(m) for m in mass]
+    >>> pdf_plot = plt.plot(mass, pdf)
+    
+    >>> samples = BW.rvs(1000)
+    >>> hist_plot = plt.hist(samples, normed=True, bins=100, range=[0, 200])
+    
+    >>> plt.show()
+    
+    Statistics of a Breit-Wigner distribution
+    -----------------------------------------
+    
+    Mean and variance:
+    
+    >>> BW.stats()
+    (array(122.11538219739913), array(762.7536855273247))
+    
+    Median:
+    
+    >>> BW.median()
+    124.41711775583536
+    
+    Variance and standard deviation:
+    
+    >>> BW.var(), BW.std()
+    (762.75368552732471, 27.617995682658158)
+    
+    95 per cent interval:
+    
+    >>> BW.interval(0.95)
+    (53.719690871945822, 159.27077425799101)
+    
+    Moments:
+    
+    >>> [BW.moment(n) for n in range(1, 6)]
+    [122.11538219739913, 15674.920254744189, inf, 244138132.80994478, 80583978992.641495]
+    
+    """
```

Comment:
Have a look at the docstrings of other distributions, they have magic substitution keys (compare to online docs, where the keys are expanded). I probably makes sense to reuse the standard format.